docs/_includes/footer.html

```diff
@@ -6,30 +6,20 @@
 			</div>
 			<div class="col-md-8 offset-md-1">
 				<div class="row">
-						<div class="col-md-4 col-xs-12">
```

Comment:
don't we want to still keep the docs links in the footer, but just use the full URL of the docs page?

Reply:
I will add it back and change links once we the docs site is live